client/src/Components/ShippingInformation.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+
+class ShippingInformation extends React.Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      shippingTime: [],
+      exclusions: [],
+      returnPolicy : [],
+      returnOverview : [],
+      giftWrapping: [],
+      getShippingCost: false,
+      shippingCost: [],
+      enteredShippingInfo: false,
+      zipCode : [],
+      shippingFrom: [],
+      shippingTo: [],
+      shippingToCountry: [],
+      deliveryDate : []
+    }
+    this.getShippingInformation = this.getShippingInformation.bind(this);
+    this.getReturnPolicy = this.getReturnPolicy.bind(this);
+    this.getShippingInfo = this.getShippingInfo.bind(this);
+    this.showShipping = this.showShipping.bind(this);
+    this.shippingCost = this.shippingCost.bind(this);
+    this.shippingToCountry = this.shippingToCountry.bind(this);    
+  }
+
+
+  componentDidMount(){
+    this.getShippingInformation()
+    this.getReturnPolicy()      
+  }
+  getShippingInformation(){
+    let manufacturingTime;
+    let shippingFrom = this.props.data[0]['shippingFrom']
+    let shippingTo = this.props.data[0]['shippingTo']
+    let deliveryDate = this.props.data[0]['deliveryDate']
+    manufacturingTime = this.props.data[0]['manufacturingTime'];
+    this.setState({
+      shippingTime: manufacturingTime,
+      shippingFrom: shippingFrom,
+      shippingTo: shippingTo,
+      deliveryDate : deliveryDate
+    })
+  }
+
+  getReturnPolicy(){
+    var returnable;
+    var returnPolicy ="";
+    var exclusion;
+    var wrapping = "";
+    if (this.props.data[0]['availableToReturn']){
+      returnable = "Returns and exchanges accepted"
+      exclusion = "Exceptions may apply. "
+      returnPolicy ="See return policy"
+      wrapping = "Gift wrapping available"
+    } else{
+      returnable ="No returns or exchanges"
+      exclusion = "But please contact me if you have any problems with your order."
+    }
+    this.setState({
```

Comment:
the further i'm getting, the harder it is to follow along with what state variables you have and where they are used/not used.  My general concern is that you have at least some state variables declared twice (i.e. you have `      zipCode` defined in state in both `overview` and this module (shipping).  There are a few more state variables that are duplicated.

If you change state in a submodule, it won't trigger a render() of the parent module(s), as far as I know.  It can also result in having state vary between modules